test-framework/core/src/main/java/org/keycloak/testframework/util/ApiUtil.java

```diff
@@ -7,9 +7,13 @@ public class ApiUtil {
 
     public static String handleCreatedResponse(Response response) {
         try (response) {
-            String uuid = getCreatedId(response);
-            response.close();
-            return uuid;
+            if (response.getStatus() != Response.Status.CONFLICT.getStatusCode()) {
+                String uuid = getCreatedId(response);
+                response.close();
```

Comment:
It's a bit odd now that it explicitly calls response.close if not 409, but skips if not. Actually no need to call close at all since it's wrapped in `try (response)`, so can just remove this line
```suggestion
```